src/accounts.js

```diff
@@ -340,12 +340,15 @@ function eosBase32(base32String) {
 }
 
 // Recursively generates account names, until a uniq name is generated...
-async function generateAccountName(prefix = '') {
+async function generateAccountName(prefix = '', ignoreExistingName = false) {
```

Comment:
Can we rename this to something else that is more clear? Like dontCheckIfNameInUseOnChain